db/schema.rb

```diff
@@ -11,26 +11,26 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20200207213514) do
+ActiveRecord::Schema.define(version: 20200720140244) do
 
   create_table "branches", force: :cascade do |t|
     t.datetime "git_updated_at",              null: false
-    t.text     "name",           limit: 1024, null: false
+    t.text     "name",           limit: 3072, null: false
```

Comment:
I was never clear on where the 3x came from in `text` sizes. Does this indicate that we added it? It's always been a giant PITA. I wonder about backing away from it in our hobo_field 3.x? Especially since 3x is antiquated and UTF-8 can, worst case, be 4x.

If we were to back away, we'd clarify that the given size is the number of bytes, not the worst-case number of characters. I think that would be fine for most of our code.

Reply:
I'm not entirely sure either.  How would you feel about creating a separate ticket and adding it to the `DeclarativeSchema` epic to track this down and set it more properly?

Reply:
The odd thing is that we're going to run a migration that grows this by 3x...then do a separate ticket that undoes that.

Hmm, I was afraid of this: apparently I added the `* 3` 5 years ago: https://github.com/Invoca/hobo_fields/commit/16df6bd227f640ea42b61560c7e3a87fe04cc7ef

I think we had some bugs where our text fields didn't have the space we thought they'd have.

Does this 3x popping up here mean that `master` of pre_deploy_checker isn't using out fork of `hobo_fields`? Or a version earlier than 2015?

Reply:
It was using a version much older than `2015`.  This repo's dependencies weren't updated very often after it was created.  Thats why also all the `null: false` defaults are suddenly kicking in, because I think that was added in 2016 or sometime around there.

Reply:
I guess we can just put up with the 3x getting in there for now. It won't actually matter since TEXT fields are always stored in MySQL at either 0xff, 0xffff, 0xffffff, or 0xffffffff bytes.

I will add a ticket to the Hobofields epic to straighten this `text` mess out one way or another!